packages/mason_logger/lib/src/mason_logger.dart

```diff
@@ -49,6 +52,21 @@ class Logger {
     _stdout.writeln(message);
   }
 
+  /// Writes a message with a clickable link.
+  /// By default writes to `stdout` with [info], but any [StdoutWriter] works.
+  ///
+  /// **NOTE**: Only works on supported terminal emulators. Please see this
+  /// [link on what is supported](https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda#supporting-apps).
+  void link(
```

Comment:
Do we need this? Can't we just interpolate `linkWrap` into existing messages?

Reply:
We could remove it and leave the top-level `linkWrap` function

Reply:
That's my preference in an attempt to keep things simple if it's alright with you.

Reply:
Totally fine with me :)